src/Nox.ClientApi.Tests/.nox/design/Entitities/store.entity.nox.yaml

```diff
@@ -11,13 +11,8 @@ description: Stores
 keys:
 
   - name: Id
+    type: databaseGuid
     isRequired: true
-    description: NuidField Type
```

Comment:
why the change, this way we lose the tests regarding Nuid?

Reply:
We already have workplace.entity.nox.yaml with the same nuid key and configuration. 
![image](https://github.com/NoxOrg/Nox.Generator/assets/30667828/8a8b5845-9bb3-4cb2-b392-1c9a500d386f)

And if I saw correctly, it has Nuid tests.
![image](https://github.com/NoxOrg/Nox.Generator/assets/30667828/f983f46b-e5ac-4052-8088-b7d0e754d70a)

Therefore I decided to change the Store key to databaseGuid.